webapp/actions/websocket_actions.jsx

```diff
@@ -237,13 +245,18 @@ function handleLeaveTeamEvent(msg) {
 
         // if they are on the team being removed redirect them to the root
         if (TeamStore.getCurrentId() === msg.data.team_id) {
+            BrowserStore.removeGlobalItem('team');
```

Comment:
If we are going to move this to the preferences then this will have to change eventually